framework/source/class/qx/test/Mixin.js

```diff
@@ -240,8 +240,8 @@ qx.Class.define("qx.test.Mixin",
           {
             var s = "";
             if (!dontRecurs) {
-              this.__b = new qx.Patch2();
-              s += "++" + this.__b.foo(true) + "__";
+              this['__b'] = new qx.Patch2();
```

Comment:
This looks like it's trying to fix a "__b is not a member" problem because __b is not declared in the members section?